src/Groups/GAPGroups.jl

```diff
@@ -246,7 +246,7 @@ relatively quickly converges to a uniform distribution.
 
 """
 function rand_pseudo(G::GAPGroup; radius::Int = 10)
-  return group_element(G, GAP.Globals.PseudoRandom(G.X; radius = radius))
+  return group_element(G, GAP.Globals.PseudoRandom(G.X; radius = radius)::GapObj)
```

Comment:
@CameronBraunstein can you please add an example to the `rand_pseudo` docstring, so that it gets coverage?